### PR TITLE
feat: add support for sha256 hashing

### PR DESCRIPTION
**Pull request by ujjwal-ab**
## Description of the change

Add function for hashing to sha256 in common bindings

### Development

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ] This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: src/bindings/common.ts [hunk 1]
```diff
@@ -56,3 +57,10 @@ export function toSeconds(timestamp: string): number | undefined {
   }
   return Math.floor(timeInMillis / 1000);
 }
+
+export function toSHA256(text: string | number | undefined) {
```
[the remark this answers is not in the document]
Comment (ujjwal-ab) [reply src/bindings/common.ts]: Done

File: src/bindings/common.ts [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import { at, identity } from 'lodash';
 import { ReturnResultError } from '../steps';
 import { StatusError } from '../common';
+import sha256 from 'sha256';
```
[the remark this answers is not in the document]
Comment (ujjwal-ab) [reply src/bindings/common.ts]: Updated

File: src/bindings/common.ts [hunk 3]
```diff
@@ -56,3 +57,14 @@ export function toSeconds(timestamp: string): number | undefined {
   }
   return Math.floor(timeInMillis / 1000);
 }
+
+export async function SHA256(text: string | number | undefined) {
```
Comment (koladilip) [inline src/bindings/common.ts:61]: Make it sync, using digestSync

**Review by koladilip — CHANGES_REQUESTED** [review]

File: src/bindings/common.ts (hunk 3 above)
Comment (ujjwal-ab) [reply src/bindings/common.ts]: Done

**Review by koladilip — APPROVED** [review]

**Review by sanpj2292 — APPROVED** [review]

**Pull request merged**